OOP 1 Homework/src/com/company/Sorter.java

```diff
@@ -1,7 +1,57 @@
 package com.company;
 
 
+import java.util.Scanner;
+
+public class Sorter {
+    Scanner intSc = new Scanner(System.in);
+    Scanner stringSc = new Scanner(System.in);
+
+    public Person[] populateArray() {
```

Comment:
This method is not related to the Sorter class' scope/responsibility.
Also it makes more sense to have it as a static method, since it does not alter the state of any object.